packages/nextjs/utils/scaffold-stark/types.ts

```diff
@@ -17,10 +17,10 @@ import type {
 import { AbiEnum, AbiStruct } from "./contract";
 
 export const isCairoInt = (type: string): type is CairoInt =>
-  /^core::integer::(u|i)(8|16|32)$/.test(type);
+  /core::integer::(u|i)(8|16|32)$/.test(type);
```

Comment:
what is the difference if we remove the `^`

Reply:
the type string passed in here may have some prefix(e.g. function name prefix `echo_int_intput_u16_core::integer::u16`),  removing`^` will only check the end part